src/sagas/lists.js

```diff
@@ -47,45 +52,33 @@ function* fetchList(action) {
   }
 }
 
-function* fetchListLessons(action) {
-  const { withViews = false, namespace, ...args } = action.payload;
```

Comment:
withViews is not relevant? You never use it?

Reply:
yes, not need, all units are with views